src/action.js

```diff
@@ -49,16 +50,16 @@ module.exports = async function run({
     const pr = pull_request || (await client.getPullRequest(PR_NUMBER))
 
     const isDependabotPR = pr.user.login === dependabotAuthor
-    if (!isDependabotPR) {
+    if (!isDependabotPR && !SKIP_VERIFICATION) {
```

Comment:
```suggestion
    if (!SKIP_VERIFICATION && !isDependabotPR) {
```

Place variables in the front for fast return.